src/main/java/uk/gov/ch/repository/psc/PersonsWithSignificantControlRepository.java

```diff
@@ -77,7 +77,28 @@ public interface PersonsWithSignificantControlRepository extends PagingAndSortin
                     + "left outer join nature_of_control_type noct on NOCT.NATURE_OF_CONTROL_TYPE_ID=NOC.NATURE_OF_CONTROL_TYPE_ID "
                     + "left outer join CORPORATE_APPT_DETAIL CAD on CAD.CORPORATE_APPT_DETAIL_ID=CBA.CORPORATE_APPT_DETAIL_ID "
                     + "where CBA.CORPORATE_BODY_ID = (select CORPORATE_BODY_ID from CORPORATE_BODY where INCORPORATION_NUMBER = ?1) "
-                    + "AND CBA.RESIGNATION_IND = 'N' AND CBA.SUPER_SECURE_PSC_IND = 'N' AND CBA.APPOINTMENT_TYPE_ID IN (5007, 5008, 5009)",
+                    + "AND CBA.RESIGNATION_IND = 'N' AND CBA.SUPER_SECURE_PSC_IND = 'N' AND CBA.APPOINTMENT_TYPE_ID IN (5010, 5011, 5012)",
             nativeQuery = true)
     Page<PersonWithSignificantControl> findPersonsWithSignificantControl(String companyNumber, Pageable pageable);
+
+    @Query(value = "select "
+            + "CBA.CORPORATE_BODY_APPOINTMENT_ID, "
+            + "CBA.APPOINTMENT_DATE, "
```

Comment:
don't need appointment date.

Reply:
The ticket mentions that appointed date is needed- should i update this to reflect its not needed too? https://companieshouse.atlassian.net/jira/software/c/projects/UAR/boards/481?modal=detail&selectedIssue=UAR-407

Reply:
I'm ok with including it - which is what you have done here